src/ui/ConnectToServer/ConnectToServer.scss

```diff
@@ -70,7 +70,7 @@
     top: -$spacer;
 
     &__Btn {
-      margin: 0 ($spacer / 4);
+      margin: 0 (calc($spacer / 4));
```

Comment:
Using [calc](https://developer.mozilla.org/en-US/docs/Web/CSS/calc) makes this compute when evaluating the CSS rather than when compiling the SASS and adds verbosity. What's the context on these changes?

Reply:
It's related to this: https://sass-lang.com/documentation/breaking-changes/slash-div/
`math.div` may be a more equivalent change, it just required imports.
Actually this let me notice that they have an automatic migrator, so I could run that and see what the output is

Reply:
`sass` does actually simplify `calc` expressions to a single number as well.